File: CarRental/Models/Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CarRent.Models
{
	public class Order
	{
		[Key]
		public int Id { get; set; }
		[Required]
		public string Title { get; set; }
		[Required]
		public DateTime StartRent { get; set; }
		[Required]
		public DateTime EndRent { get; set; }

		public string Discription { get; set; }
		
		public int UserId { get; set; }
		public User User { get; set; }
		public int CarId { get; set; }
		public Car Car { get; set; }

	}
}

File: CarRental/ClientApp/app/Order.ts
import { User } from "./user";
import { Car } from "./car";

export class Order {
	constructor(
		public id?: number,
		public title?: string,
		public startRent?: Date,
		public endRent?: Date,
		public discription?: string,
		public userId?: number,
		public user?: User,
		public carId?: number,
		public car?:Car

	){ }
}